devtools/aesh/src/main/java/io/quarkus/cli/commands/AddExtensionCommand.java

```diff
@@ -44,7 +44,7 @@ public CommandResult execute(CommandInvocation commandInvocation) throws Command
                 try {
                     AddExtensions project = new AddExtensions(new File(pom.getAbsolutePath()));
                     project.addExtensions(Collections.singleton(extension));
-                } catch (IOException e) {
+                } catch (IOException | CommandsException e) {
```

Comment:
I would catch all exceptions here and I would probably return a status different from `CommandResult.SUCCESS`.

Note that it's not related to what you did, the current code is wrong IMHO.